backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -94,24 +94,9 @@ export class AdminResolver {
     result.creation = await getUserCreations(user.id)
 
     return result
-
-    // const creations = await getUserCreations(user.id)
-    // const creationDateObj = new Date(creationDate)
-    // if (isCreationValid(creations, amount, creationDateObj)) {
-    //   const pendingCreationRepository = getCustomRepository(PendingCreationRepository)
-    //   const loginPendingTaskAdmin = pendingCreationRepository.create()
-    //   loginPendingTaskAdmin.userId = user.id
-    //   loginPendingTaskAdmin.amount = BigInt(amount * 10000)
-    //   loginPendingTaskAdmin.created = new Date()
-    //   loginPendingTaskAdmin.date = creationDateObj
-    //   loginPendingTaskAdmin.memo = memo
-    //   loginPendingTaskAdmin.moderator = moderator
-    //
-    //   pendingCreationRepository.save(loginPendingTaskAdmin)
-    // }
-    // return await getUserCreations(user.id)
   }
 
+  @Authorized([RIGHTS.SEARCH_USERS])
```

Comment:
This seems to be the wrong right - every function gets its own right.

Reply:
OK, I was not sure while implementing it, I understood that SEARCH_USERS is the actual Admin role / right